keyboards/cannonkeys/satisfaction75/rules.mk

```diff
@@ -54,4 +54,4 @@ QWIIC_ENABLE += MICRO_OLED
 # RAW_ENABLE = yes
 # DYNAMIC_KEYMAP_ENABLE = yes
 
-DEFAULT_FOLDER = cannonkeys/satisfaction75
+DEFAULT_FOLDER = cannonkeys/satisfaction75/rev1
```

Comment:
I'm actually not a fan of default folders. I prefer users to have to explicitly compile the keyboard AND revision they want.

Reply:
I will take it out